.github/workflows/compose-maven-verify.yaml

```diff
@@ -115,6 +115,7 @@ jobs:
       - uses: lfit/checkout-gerrit-change-action@54d751e8bd167bc91f7d665dabe33fae87aaaa63 # v0.9
         with:
           gerrit-refspec: ${{ inputs.GERRIT_REFSPEC }}
+          gerrit-project: ${{ inputs.GERRIT_PROJECT }}
```

Comment:
I've reverted this from ae623e78b0a5 since this retuns an error.

   Error:
    Error: Invalid repository 'controller'. Expected format {owner}/{repo}.

Should this be  "${{ vars.$ORG}}/${{ inputs.GERRIT_PROJECT }}"?